includes/amp/class-image-list.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Class Image_List
+ *
+ * @package AMP
+ */
+
+namespace AMP;
+
+/**
```

Comment:
Again, I suggest importing all external dependencies, as that makes it very clear what to watch out for later when refactoring or diagnosing bugs.
```suggestion
use ArrayIterator;
use Countable;
use DOMElement;
use IteratorAggregate;

/**
```

Reply:
Ah, good idea. https://github.com/ampproject/amp-wp/pull/3659/commits/3f0ee523069fcda4e8588f71d781256dad3346ca imports the classes. 

Your commit above looks good, but I moved the files to `src/component/` earlier. I should have committed that suggestion first 😄